backend-client/openapi-generator.sh

```diff
@@ -20,7 +20,7 @@ function clean_up() {
 }
 
 function generate_client_docker() {
-  docker run --rm -v ${PWD}:/local -u `id -u`:`id -g` openapitools/openapi-generator-cli generate \
+  docker run --rm -v "${PWD}":/local -u "$(id -u)":"$(id -g)" openapitools/openapi-generator-cli generate \
```

Comment:
We don't need the curly braces!
```suggestion
  docker run --rm -v "$PWD":/local -u "$(id -u)":"$(id -g)" openapitools/openapi-generator-cli generate \
```